spec/src/main/asciidoc/servlet-spec-body.adoc

```diff
@@ -1371,9 +1336,7 @@ with an attribute name.
 
 Attribute names beginning with the prefix of
 `jakarta.` are reserved for definition by this specification.
```

Comment:
Isn't "this specification" only going to define `jakarta.servlet` attributes?

Reply:
As far as I recall all attributes will be `jakarta.servlet.*`. That said, I don't see the harm in reserving the wider namespace.